css/at-rules/custom-media.json

```
@@ -0,0 +1,43 @@
+{
+  "css": {
+    "at-rules": {
+      "custom-media": {
+        "__compat": {
+          "description": "<code>@custom-media</code>",
+          "spec_url": "https://www.w3.org/TR/mediaqueries-5/#custom-mq",
```

We usually link to the Editor drafts:

```suggestion
          "spec_url": "https://drafts.csswg.org/mediaqueries-5/#at-ruledef-custom-media",
```